packages/mesos-overlay-modules/buildinfo.json

```diff
@@ -2,8 +2,8 @@
   "requires": ["mesos"],
     "single_source" : {
       "kind": "git",
-      "git": "https://github.com/dcos/mesos-overlay-modules.git",
+      "git": "https://github.com/asridharan/mesos-overlay-modules.git",
```

Comment:
This is already merged into DC/OS master. Could you rebase and remove this commit ?